cmd/argocd/commands/admin/settings_rbac.go

```diff
@@ -359,14 +360,37 @@ func getPolicyFromFile(policyFile string) (string, string, string, error) {
 // Retrieve policy information from a ConfigMap
 func getPolicyFromConfigMap(cm *corev1.ConfigMap) (string, string, string) {
 	var (
+		strBuilder  strings.Builder
 		userPolicy  string
 		defaultRole string
+		p           string
 		ok          bool
 	)
-	userPolicy, ok = cm.Data[rbac.ConfigMapPolicyCSVKey]
-	if !ok {
-		userPolicy = ""
+
+	// add the main policy first
+	if p, ok = cm.Data[rbac.ConfigMapPolicyCSVKey]; ok {
+		strBuilder.WriteString(p)
 	}
+
+	keys := make([]string, 0, len(cm.Data))
+	for k := range cm.Data {
+		keys = append(keys, k)
+	}
+
+	// add the other policies
+	sort.Strings(keys)
+	for _, key := range keys {
+		value := cm.Data[key]
+		if strings.HasPrefix(key, "policy.") &&
+			strings.HasSuffix(key, ".csv") &&
+			key != rbac.ConfigMapPolicyCSVKey {
```

Comment:
I was wondering if this check should be done already while looping through the CM and building the list of available keys.

Reply:
If you are saying that it would be nice to do the check while creating the list, I think it would have the advantage of simplifying the process of creating the slice above, although there is a little overhead. 
And actually, I referred to other [similar code](https://github.com/argoproj/argo-cd/blob/014f4424b7218d4d98848563a5194f0492b12ba4/util/rbac/rbac.go#L406). Since there is already other code in the source tree that does the same thing, I could add the code without any burden.

Reply:
That code reference is actually a good point. I think it would be great if there would be function that could be shared, instead of having the code duplicated.

Reply:
I modified it to respect the existing code.